soroban-sdk-macros/Cargo.toml

```diff
@@ -23,7 +23,7 @@ syn = {version="2.0.77",features=["full"]}
 quote = "1.0"
 proc-macro2 = "1.0"
 itertools = "0.10.5"
-darling = "0.20.0"
+darling = "0.20.10"
```

Comment:
With the use of `darling(with = )` the sdk now requires a newer version of darling that was previously not being required.